kalliope/signals/geolocation/README.md

```diff
@@ -0,0 +1,76 @@
+# Geolocalisation
+
+## Synopsis
+
+**Geolocation** is a way to launch a synapse when ENTERING a geolocated zone.
+
+As Kalliope does not manage its own geolocation, this signal has been designed in view to be implemented from external clients (smartphones, watches, embedded devices, etc).
+
+The syntax of a geolocation declaration in a synapse is the following.
```

Comment:
could be useful for the user to know how to get his localisation, don't you think?

Reply:
:) if you want I can provide a link but ... isn't it a bit out of the kalliope scope ? 
The user can provide any geolocation ... not only its own

Reply:
Yes your are right. It's out of scope. We'll see later if we need to add stuff around this feature to explain.